src/templates/newengland-organizing.html

```diff
@@ -0,0 +1,102 @@
+                        <li>a PA system (nice to have, but not a requirement)</li>
+                        <li>a projector (even less of a requirement but still nice to have)</li>
+                        <li>tables (rectangular competition tables are a must (or you’ll need to rent some), and other tables for the audience and competitors to sit at are nice)</li>
+                        <li>lighting that is appropriate for solving that will be relatively consistent throughout the day</li>
+
+                </ul>
+                <p>When you start writing up a budget, you should have a good idea of how much the venue will cost. Schools and universities can often provide low-cost venues, but that’s not always an option. Be sure to make note of required custodians or police officers, as this usually costs extra. Places of worship like churches may be good candidates, but their space is often either too small or not suitable (ie full of pews). Community centers and similar places are also good venues to try.</p>
+                <h3 id="pa-system-wifi">PA System, WiFi</h3>
+                <p>A PA system is really a must. If the venue doesn’t have one, Matthew McMillan has one he may be able to bring, but he needs to know this in advance. It’s very difficult to have a competition where all announcements are made by shouting. Similarly, it’s possible to have a competition without WiFi, but it’s far from ideal. If this has to happen, let your Delegate know in advance.</p>
+                <h3 id="the-finances">The finances</h3>
+                <p>The organizer is responsible for creating a budget, though the Delegate will need to approve it. The organizer will usually need to create a Stripe account to collect the registration fees and link this to the WCA website, but I’m happy to do this if that poses a problem. Please fill out the budget tab in the spreadsheet linked to at the top of this document when it’s time to make a budget. Note that you should plan for registration income to cover printing costs, battery costs, and any other costs related to the competition while still leaving a safe profit margin. We generally prefer to use final profits from a competition to purchase future equipment for the region.</p>
+                <h3 id="deciding-on-events">Deciding on events</h3>
+                <p>You definitely have some say over what events you want, but this competition should benefit the region too. Holding at least some beginner events, like 3x3, 2x2, and Pyraminx allows beginners to compete which is good for the community. It’s fine if you want to have less common events like 7x7, but the competition should be fun for everyone. In terms of scheduling, you should create a tentative schedule based on what you think is realistic. It’s good to add buffer time in case some events get behind. The Delegate will help you adjust the schedule to be more reasonable if needed.</p>
```

Comment:
"Work with the delegate to select the events for the competition.  In general, at most competitions we recommend holding a few events like 3x3x3, 2x2x2, and Pyraminx that are popular with a large number of competitors"